### PR TITLE
Feat: add l2 chain info

### PR DESCRIPTION
**Pull request by moojing**
## Summary

- add chain info of  arbitrum and optimism.
 
## Related Links

**Asana**:  

**Mockup**:  

## Checklist

- [ ] Pasted Asana or Mockup link.
- [ ] Tagged labels.

## Prerequisite/Related Pull Requests

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204200709519569
  - https://app.asana.com/0/0/1204343719537803

### CODE REVIEW

File: src/services/arbitrum.ts [hunk 1]
```diff
@@ -6,11 +6,10 @@ const isMainnet = process.env.REACT_APP_NETWORK === "mainnet";
 
 const bloctoSDK = new BloctoSDK({
   ethereum: {
-    server: "http://localhost:3000",
     chainId: isMainnet ? "0xa4b1" : "0x66eed",
     rpc: isMainnet
-      ? "https://endpoints.omniatech.io/v1/arbitrum/one/public"
-      : "https://endpoints.omniatech.io/v1/arbitrum/goerli/public",
+      ? "https://arb1.arbitrum.io/rpc"
+      : "https://goerli-rollup.arbitrum.io/rpc",
```
Comment (moojing) [inline src/services/arbitrum.ts:12]: do you encounter any problem with old rpc endpont?

File: src/services/optimism.ts [hunk 2]
```diff
@@ -6,7 +6,6 @@ const isMainnet = process.env.REACT_APP_NETWORK === "mainnet";
 
 const bloctoSDK = new BloctoSDK({
   ethereum: {
-    server: "http://localhost:3000",
```
Comment (moojing) [inline src/services/optimism.ts]: my fault lol

File: src/services/arbitrum.ts (hunk 1 above)
Comment (akira02) [reply src/services/arbitrum.ts:12]: BE not add in white list, so use official instead

File: src/services/rinkeby.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ const isMainnet = process.env.REACT_APP_NETWORK === "mainnet";
 
 const bloctoSDK = new BloctoSDK({
   ethereum: {
+    server: "http://localhost:3000",
```
[the remark this answers is not in the document]
Comment (akira02) [reply src/services/rinkeby.ts:16]: Thanks I missed this file

**Review by mordochi — APPROVED** [review]

File: yarn.lock [hunk 4]
```diff
@@ -1301,10 +1301,10 @@
...
 
-"@blocto/sdk@^0.4.0":
-  version "0.4.0"
-  resolved "https://registry.yarnpkg.com/@blocto/sdk/-/sdk-0.4.0.tgz#5f11b3c22a90e8c68b08c09898f21ce8690411b2"
-  integrity sha512-ohDaIw5gdDDWoudu7/Zl+//XB+5SafpIPa6bmaTfUzbaq7iD/ad3ZY9PnyirBj2QxKFeJV7+DzNRdZ+XN6EgcA==
+"@blocto/sdk@0.4.2-beta.1":
+  version "0.4.2-beta.1"
+  resolved "https://registry.yarnpkg.com/@blocto/sdk/-/sdk-0.4.2-beta.1.tgz#8ff58f8bfa88919938cef43a70db90a18b5cbd19"
+  integrity sha512-E8TthQ0+RbdrfzXomr6YHQP4xtq08pCbOi5p7PPymJapBZP/qhAQ91Nq/bS1wTA94hX7h65AhKAC16LbzLzmfw==
   dependencies:
```
[the remark this answers is not in the document]
Comment (akira02) [reply yarn.lock:1308]: That's weird. I've run yarn once. Anyway I checked should be okay.

**Pull request merged**